src/Features/Core/Portable/EditAndContinue/EditAndContinueWorkspaceService.cs

```diff
@@ -153,7 +153,7 @@ internal static bool SupportsEditAndContinue(Project project)
             => project.LanguageServices.GetService<IEditAndContinueAnalyzer>() != null;
 
         internal static bool IsDesignTimeOnlyDocument(Document document)
```

Comment:
Do we need this extension method anymore? Or can we just inline the code? I always hate these sort of methods because it makes finding who actually depends on a piece of data harder to work out since it hides uses behind a method.